### ISSUE TITLE
Login required after container restarts using HAPPYDOMAIN_NO_AUTH

### ISSUE THREAD

**Issue by systemcrash** [happyDomain/happydomain#8]
First time run with HAPPYDOMAIN_NO_AUTH. 
Restart container with same flag HAPPYDOMAIN_NO_AUTH

Surf to URL. Login required. 

User must manually destroy cookies made by HD.  ( cookies, uh-oh.... )

**Comment by JS3910** [comment]
Yeah I thought this looked like a nice and useful project, but it seems all the talk of "libre", "free", "independent", etc, etc everywhere on their website is nonsense. After adding a DNS provider the software doorslams hard whatever you do and your only choice apparently is to sign up for an account on their cloud service.

**Comment by systemcrash** [comment]
Only after testing and firewalling my instance was I able to verify that
any account you set up in your local instance remains local and no info is
shared.

This was not evident in the documentation.

In any case using their service might make life easier.

**Comment by nemunaire** [comment]
> Yeah I thought this looked like a nice and useful project, but it seems all the talk of "libre", "free", "independent", etc, etc everywhere on their website is nonsense. After adding a DNS provider the software doorslams hard whatever you do and your only choice apparently is to sign up for an account on their cloud service.

Hi @JS3910,

Thanks for reaching us.

You certainly faced with a bug on the DNS provider you tried to use on your local instance, and the same bug should exist on our main instance until somebody report it (we don't have accounts on all providers we offer, as we rely on [an underlying project](https://github.com/StackExchange/dnscontrol) for that).

Please open a new bug report, we will investigate with you what happen with your DNS provider. Perhaps we also need more documentation? or to advertise better on how to deploy the service? in any case we are listening and open to suggestion in bug reports.

This repository held the entire codebase of the service available on https://www.happydomain.org/, you can deploy the exact same service (without the showcase site, that is [on another repo](https://git.happydomain.org/website)) with the [Docker container](https://help.happydomain.org/en/deploy/docker/) or [the binary](https://get.happydomain.org/master/) (or even compile [the source code](https://github.com/happyDomain/happydomain#building)). The automation recipes for Ansible are also provided.

We even don't have any ads, cookie banner, pay-wall, nor paying offer at this time on our cloud based service. We're certainly no good at marketing our service, also if you have some tips to avoid the negative feeling you show in your message, we'll try to improve the situation.

**Comment by systemcrash** [comment]
This problem seems largely resolved as a result of changes made originating in #1. The header banner changes briefly during page reloads. The first page reload after the image is restarted results in:

first time:
![Screenshot 2023-09-14 at 16 37 39](https://github.com/happyDomain/happydomain/assets/647633/e4fb7eaa-86c4-42bd-9ef9-a028475727d1)

thereafter:
![Screenshot 2023-09-14 at 16 37 35](https://github.com/happyDomain/happydomain/assets/647633/45b2fd9b-4845-45ef-91af-ab2caf4fa8e8)

I don't know whether it's possible to fix, and this seems minor, but might trigger suspicion among the paranoid and security conscious. 

I evaluated Container ID 80e2bc0ebe2e of happydomain/happydomain
